tests/components/blue_current/test_button.py

```diff
@@ -0,0 +1,52 @@
+"""The tests for Blue Current buttons."""
+
+import pytest
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import init_integration
+
+from tests.common import MockConfigEntry
+
+charge_point_buttons = ["stop_charge_session", "reset", "reboot"]
+
+
+async def test_buttons_created(
+    hass: HomeAssistant, config_entry: MockConfigEntry
+) -> None:
+    """Test if all buttons are created."""
+    await init_integration(hass, config_entry, "button")
+
+    entity_registry = er.async_get(hass)
+
+    buttons = er.async_entries_for_config_entry(entity_registry, "uuid")
+    assert len(charge_point_buttons) == len(buttons)
```

Comment:
This can be replaced with a snapshot_platform helper, after this the unique_id check in the next test can be removed